### PR TITLE
Calendar for Penn Mobile Server

### PR DESCRIPTION
**Pull request by esqu1**

### CODE REVIEW

**Comment by adelq** [comment]
Scraping is always a last resort...why not use the export function to get easily parsed XML or iCal? http://www.upenn.edu/almanac/acadcal.html

**Comment by esqu1** [comment]
The XML link doesn't work on the website.

**Comment by adelq** [comment]
Looks like Google Calendar discontinued XML export, but the ICS still works perfectly and is much more specialized to calendars anyways.

File: penn/calendar3year.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                end_date = datetime.datetime.strptime(raw_date,'%Y%m%d').date()
+                d['end'] = end_date
+            elif line[:7] == "SUMMARY":
+                name = line.split(":")[1]
+                d['name'] = name.encode('utf-8').strip()
+            elif line == "END:VEVENT":
+                events.append(d)
+
+        events.sort(key=lambda d: d['start'])
+        print events
```
Comment (adelq) [inline penn/calendar3year.py]: No print

Comment (esqu1) [inline penn/calendar3year.py]: Every time...

**Comment by adelq** [comment]
LGTM

**Pull request merged**